Code/Source/SceneConfigurationComponent.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <AzCore/Component/Component.h>
+
+namespace RGL
+{
+    struct SceneConfiguration
+    {
+        AZ_TYPE_INFO(SceneConfiguration, "{7e55de90-e26c-4567-9e06-822c6ce62b9c}");
+        static void Reflect(AZ::ReflectContext* context);
+
+        bool m_isSkinnedMeshUpdateEnabled{ true };
+    };
+
+    class SceneConfigurationComponent : public AZ::Component
```

Comment:
I would argue that since this name is inside of the RGL namespace, the name may be generic. Additionally the name visible inside of the Editor (from the user perspective) has "RGL" in it therefore can be easily distinguish. I would avoid prefixing the class / struct name when it is already contained within a namespace.